### PR TITLE
Remove the splash screen (to save 100kb (compressed!))

### PR DESCRIPTION
**Pull request by DHowett**
We're a Centennial application; we can't even _use_ the splash screen.

### CODE REVIEW

**Comment by DHowett** [comment]
Yes, I am deleting assets from design. I'm not worried about it, because if we _do_ need a splash screen we'll know where to find it. History.

**Comment by DHowett** [comment]
365kb is approximate, partly because I used a different compressor. Let me try again.

**Comment by DHowett** [comment]
Using makeappx cut the savings to 100kb. Still worth doing so we don't pollute the disk with a bunch of useless files :)

**Review by zadjii-msft — APPROVED** [review]
k

**Review by miniksa — APPROVED** [review]
Yep ok. Works for me.
